src/vs/editor/common/config/commonEditorConfig.ts

```diff
@@ -502,6 +502,11 @@ const editorConfiguration: IConfigurationNode = {
 			default: true,
 			description: nls.localize('wordBasedSuggestions', "Controls whether completions should be computed based on words in the document.")
 		},
+		'editor.wordBasedSuggestionsOnlySameLanguage': {
+			type: 'boolean',
+			default: true,
+			description: nls.localize('wordBasedSuggestionsOnlySameLanguage', "Controls whether word based completions should be included from opened documents of the same language or any language.")
```

Comment:
Looking, good. I might remodel this into a enum which goes like `wordBasedSuggestionSource: 'currentDocument', 'sameLanguageDocuments',  'allDocuments'`. That will allow folks to go back to old behaviour as well